dev-libs/libuv/libuv-1.9.1.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A new platform layer for Node"
+DESCRIPTION="libuv is a multi-platform support library with a focus on \
+asynchronous I/O. It was primarily developed for use by Node.js, but it's also \
+used by Luvit, Julia, pyuv, and others."
+HOMEPAGE="http://libuv.org/"
+COPYRIGHT="2009-2015 Ryan Dahl and others"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/libuv/libuv/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a6ca9f0648973d1463f46b495ce546ddcbe7cce2f04b32e802a15539e46c57ad"
+PATCHES="libuv-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libuv$secondaryArchSuffix = $portVersion
+	lib:libuv$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libuv${secondaryArchSuffix}_devel = $portVersion
+	devel:libuv$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libuv$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	export LDFLAGS="-lroot -lnetwork -lbsd"
+	export CFLAGS="-D_BSD_SOURCE"
+
+	chmod +x autogen.sh
+	autogen.sh
```

Comment:
Should be:
```
	./autogen.sh
```
BTW, you can drop the previous line (i.e. the **`chmod +x autogen.sh`**) because the script is already executable.